workflow-templates/lint-eslint.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Linting javascript",
+    "description": "Javascript linting. Require proper script in package.json",
```

Comment:
what is proper?

Reply:
That is the question!
Same for the composer entries.
Should we have the script hardcoded here or rely on the composer.json?

Reply:
I guess we don't/can't have templates for what to have in the XXX.json configs, do you have any idea on how to approach this?

Reply:
https://github.com/nextcloud/documentation/pull/6930